test/README.md

```diff
@@ -0,0 +1,59 @@
+# contra-env-setup CI
+This document describes how works the tests in the contra-env-setup repository.
+
+## Avocado
+We are using the avocado-framework to create and run our test of integrity in contra-env-setup. 
+
+### How to install
+Into the directory, `test/` have the file `requirements.txt` file to define what version we are using in our tests.
+
+```
+$ git clone https://github.com/CentOS-PaaS-SIG/contra-env-setup.git
+$ cd contra-env-setup/test/
+$ pip install -r requirements.txt --user
+```
+
+**NOTE** this version have support to `qemu-img` that is necessary to our tests.
+
+Should install the packages are requirements:
+```
+$ sudo dnf install qemu qemu-img genisoimage python2-aexpect
+```
+
+To support `qemu` in the avocado we need install this plugin:
+```
+$ git clone https://github.com/apahim/avocado_qemu.git
+$ cd avocado_qemu
+$ pip install . --user
```

Comment:
I published the package on pip already, so this section can be dropped in favor of including `avocado-qemu` in `requirements.txt`